kore/src/Kore/Step/Function/Evaluator.hs

```diff
@@ -308,6 +310,14 @@ maybeEvaluatePattern
         identifier' <- identifier
         Map.lookup identifier' axiomIdToEvaluator
 
+    branchTracing axiomResult resimplificationResult =
+        case identifier of
+            Nothing -> return ()
+            Just identifier' -> Profile.axiomBranching
```

Comment:
If `axiomBranching` is not used elsewhere, would it make sense to have that function take a `Maybe AxiomIdentifier` as its first argument instead of an `AxiomIdentifier`?

Reply:
Fixed.